lib/phoenix_component/declarative.ex

```diff
@@ -904,17 +903,18 @@ defmodule Phoenix.Component.Declarative do
   end
 
   defp build_attr_doc_and_default(%{doc: doc, type: :global, opts: opts}, indent) do
-    case Keyword.fetch(opts, :include) do
-      {:ok, [_ | _] = inc} ->
-        if doc do
-          [build_doc(doc, indent, true), "Supports all globals plus: ", build_literal(inc), "."]
-        else
-          ["Supports all globals plus: ", build_literal(inc), "."]
-        end
-
-      _ ->
-        if doc, do: [build_doc(doc, indent, false)], else: []
-    end
+    [
+      # always included
+      "\n* Global attributes are accepted.",
+      if(doc, do: [" ", build_doc(doc, indent, false)], else: []),
+      case Keyword.get(opts, :include) do
+        inc when is_list(inc) and inc != [] ->
+          [" ", "Supports all globals plus:", " ", build_literal(inc), "."]
+
+        _ ->
+          []
```

Comment:
We start off with the generic line, then maybe add docs, then maybe add includes.

Reply:
Perfect :heart: